src/OrbitGl/TrackManager.cpp

```diff
@@ -448,7 +452,11 @@ SchedulerTrack* TrackManager::GetOrCreateSchedulerTrack() {
 }
 
 ThreadTrack* TrackManager::GetOrCreateThreadTrack(uint32_t tid) {
-  std::lock_guard<std::recursive_mutex> lock(mutex_);
+  absl::MutexLock lock(&mutex_);
```

Comment:
Nit: It's good practice to use `WriterMutexLock` instead of `MutexLock` in code that also uses `ReaderMutexLock` to make the reader aware that the very same mutex might be locked in a shared way (even though `MutexLock` and `WriterMutexLock` are both exclusive and mean the same.)

Reply:
done!